examples/samsung/aot_compiler.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2025 Samsung Electronics Co. LTD
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import logging
+
+from executorch.backends.samsung.serialization.compile_options import (
+    gen_samsung_backend_compile_spec,
+)
+from executorch.backends.samsung.utils.export_utils import (
+    to_edge_transform_and_lower_to_enn,
```

Comment:
this function `to_edge_transform_and_lower_to_enn` seems like a convenience function, right? And this file aot_compiler.py is really a binary so probably want to call it aot_compile.py